### PR TITLE
fix and pull laboratory image

### PR DESCRIPTION
**Pull request by Ronserruya**
The ENV variables passed by docker-compose don't get passed while building the "dist" folder, so now I edit the final app.js file when starting the container

The final image was uploaded to dockerhub under kinecosystem/laboratory:1

### CODE REVIEW

File: image/docker-compose.yml [hunk 1]
```diff
@@ -110,10 +110,7 @@ services:
         max-file: "3"
 
   laboratory:
-    image: kinecosystem/laboratory
-    build:
-      dockerfile: Dockerfile.laboratory
-      context: .
+    image: kinecosystem/laboratory:1
```
Comment (oryband) [inline image/docker-compose.yml]: why is this tagged `:1`? laboratory doesn't have version 1 and it's misleading to use the version tag for that. just use  `:latest`

Comment (oryband) [reply image/docker-compose.yml]: don't forget to remove the dockerhub image tagged `:1`

**Comment by oryband** [comment]
also, i think init.sh need to be updated as well:
```bash
HORIZON_ENDPOINT_TESTNET="$HORIZON_ENDPOINT_TESTNET" X=Y A=B C=D ... sudo -E docker-compose up -d laboratory
```

File: image/docker-compose.yml [hunk 2]
```diff
@@ -110,10 +110,7 @@ services:
         max-file: "3"
 
   laboratory:
-    image: kinecosystem/laboratory
-    build:
-      dockerfile: Dockerfile.laboratory
-      context: .
+    image: kinecosystem/laboratory:4ae9ca9
```
Comment (doodyparizada) [inline image/docker-compose.yml]: Now that you removed the build option from the docker-compose, I suggest creating two targets in the Makefile `build-image` to run the `docker build` and `docker tag` commands and `push-image`, to push the latest and the commit hash tag

Comment (doodyparizada) [reply image/docker-compose.yml]: example: https://github.com/kinecosystem/marketplace-server/blob/master/Makefile#L32-L43

**Pull request merged**